sql_generators/glean_usage/templates/metrics_clients_last_seen.query.sql

```diff
@@ -41,4 +41,5 @@ FULL JOIN
   _current
 USING
   (client_id,
-    sample_id)
+    sample_id,
+    normalized_channel)
```

Comment:
Are we guaranteed that `normalized_channel` is populated? IIRC, a row with a null in any of these join fields will get dropped.

Reply:
I don't think it's guaranteed. Though looking at some selected tables (Fenix) it does not seem to be `null`. But I guess we should handle the `null` cases.